tasks/Resources/Provisioning/Hive/AWS/oc-provision.robot

```diff
@@ -0,0 +1,56 @@
+*** Settings ***
+Library           OperatingSystem
+
+*** Keywords ***
+Run AWS Configuration In Hive
+    [Arguments]    &{infrastructure_configurations}
```

Comment:
If  &{infrastructure_configurations} only has 2 values, why not just passing 2 arguments?
```
Create AWS Cluster Using Hive
[Documentation] ...
[Arguments]    ${hive_cluster_name}   ${hive_claim_name}
```